vyper/builtins/functions.py

```diff
@@ -1696,6 +1696,7 @@ class _CreateBase(BuiltinFunctionT):
     _kwargs = {
         "value": KwargSettings(UINT256_T, zero_value),
         "salt": KwargSettings(BYTES32_T, empty_value),
+        "revert_on_failure": KwargSettings(BoolT(), True, require_literal=True),
```

Comment:
we had some create builtins do unchecked create (create_minimal_proxy_to iirc) right? i think this changes the semantics then